synthtool/gcp/templates/node_library/.github/CODEOWNERS

```diff
@@ -7,10 +7,7 @@
 
 # The yoshi-nodejs team is the default owner for nodejs repositories.
```

Comment:
A few things
- this comment needs an update
- in the old line 10, the github team came before the "{{metadata" curly braces. Should it here as well?
- does this github team exist yet? if not, is that way github is yelling at us for having an error?

Reply:
this comment still stands